Cargo.toml

```diff
@@ -20,8 +20,7 @@ rand_core = "0.6"
 rand = "0.8"
 rand_chacha = "0.3.1"
 tikv-jemallocator = "0.5.4"
-# wsts = { version = "8.1", default-features = false }
-wsts = { git = "https://github.com/stacks-network/wsts.git", branch = "feat/public-sign-ids" }
+wsts = { git = "https://github.com/stacks-network/wsts.git", default-features = false, rev = "1b9589df9c1ee299423bffdf8f74298d9f4464f2" }
```

Comment:
Why `default-features = false`?

Reply:
This is how we support musl. It simplifies the p256k1 build.

Reply:
No bindgen so no need for libclang etc

Reply:
I have two answers here, depending on how to interpret the question:

1. **Why would we like to have `default-features=false` here?**
The wsts library uses bindings to the underlying `libsecp256k1` C library. It defaults to rebuilding these bindings, but Jude has added a set of precompiled bindings to avoid recompiling these. Setting `default-features=false` makes use of the precompiled `libsecp256k1` bindings instead of recompiling these.

2. **Why did I change this?**
I didn't. You did. When I began this change set, `default-features = false` was already set for the WSTS dependency - which you removed in `207cb690fe`. It did not seem intentional since that change had commented out the original dependency and pointed to a branch, so I chose to keep `default-features = false` while rebasing. If you prefer the other way around I can change this. I have no strong preference here.

Reply:
Can we avoid having a `rev` here?  Let's just publish a stable version of `wsts` with the features you need before merging this.

I don't care how you want to deal with `default-features = false`.  All I care about is that we can still build both glibc and musl Linux images.

Reply:
Yeah this isn't getting merged until I publish the new wsts crate.  But I can't do that until @jcnelson  or @kantai  approves wsts#7

Reply:
Ah, please feel free to be aggressive at pinging me when you're waiting for one of our reviews! :D

Approved :pray: